Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -141,11 +141,17 @@ protected override void SetAccessibilityHint()
 
 			if (Element == null)
 				return;
-
+#if __MOBILE__
 			if (_defaultAccessibilityHint == null)
 				_defaultAccessibilityHint = Control.AccessibilityHint;
 
 			Control.AccessibilityHint = (string)Element.GetValue(AutomationProperties.HelpTextProperty) ?? _defaultAccessibilityHint;
+#else
+			if (_defaultAccessibilityHint == null)
+				_defaultAccessibilityHint = Control.AccessibilityTitle;
+
+			Control.AccessibilityTitle = (string)Element.GetValue(AutomationProperties.HelpTextProperty) ?? _defaultAccessibilityHint;
```

Comment:
The few places in this PR where there is duplicate code like this retrieve the bindable property. Can you move these to a Re-usable property?

Something like
string HelpText => (string)Element.GetValue(AutomationProperties.HelpTextProperty) ?? _defaultAccessibilityHint;